cypress/integration/dib-travel/company-settings/travel-policy.spec.ts

```diff
@@ -0,0 +1,109 @@
+import { TravelPolicy } from '../../../models';
+
+describe('Travel Policy Suite', () => {
+  let tpd: TravelPolicy;
+
+  before(() => {
+    cy.fixture('company-settings/travel-policy').then((travelPolicyFormData) => {
+      tpd = travelPolicyFormData;
+    });
+  });
+
+  beforeEach(() => {
+    cy.login();
+    cy.get('dib-layout dib-hamburger-icon').click();
+    cy.get('[href="/company-management/travel-policy"]').click();
+  });
+
+  it('submits empty travel policy form', () => {
+    cy.get('dib-travel-policy ui-button[type=primary]').click();
+    cy.get('dib-travel-policy-dialog ui-button button').click({ force: true });
+    cy.get('dib-dialog-wrapper dib-travel-policy-dialog').should('be.visible');
+  });
+
+  it('creates a new flight travel policy', () => {
+    cy.get('dib-travel-policy ui-button[type=primary]').click();
+    cy.get('dib-travel-policy-dialog .dib-select').select(tpd.travelPolicyTypeFlight);
+    cy.get('dib-travel-policy-dialog dib-input input[name=name]').type(tpd.travelPolicyName);
+    cy.get('dib-travel-policy-dialog dib-input input[name=numberOfDaysInAdvance]').type(tpd.numberOfDaysInAdvance);
+    cy.get('dib-travel-policy-dialog dib-checkbox').click({ multiple: true });
```

Comment:
Can you provide more context, why are you using `{ multiple: true }` ?

Reply:
Because it's very practical when you have multiple checkbox in some section, you find their common/parent element and check them all at once instead of typing the code for each one. It seems good for me..?

Reply:
Nice ❤️